### ISSUE TITLE
UnboundLocalError: local variable 'chats_list' referenced before assignment

### ISSUE THREAD

**Issue by sweenzor** [carderne/signal-export#35]
Clean install in a virtualenv, produces the following error:

```
$ sigexport out
Traceback (most recent call last):
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/bin/sigexport", line 33, in <module>
    sys.exit(load_entry_point('signal-export', 'console_scripts', 'sigexport')())
  File "/Users/sweeney/src/signal-export/sigexport/main.py", line 618, in cli
    run(main)
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/typer/main.py", line 859, in run
    app()
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/sweeney/.local/share/virtualenvs/signal-export-j2I1KKmX/lib/python3.9/site-packages/typer/main.py", line 497, in wrapper
    return callback(**use_params)  # type: ignore
  File "/Users/sweeney/src/signal-export/sigexport/main.py", line 582, in main
    convos, contacts = fetch_data(db_file, key, manual=manual, chats=chats_list)
UnboundLocalError: local variable 'chats_list' referenced before assignment
```

**Comment by carderne** [comment]
Sorry! Fixed in ae24a1b2ee1e99453e9edcd9f9298284c29219be.

**Comment by sweenzor** [comment]
Thank you!
